package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "remote-settings-mdn-browser-data",
+  "version": "1.0.0",
+  "description": "Ingest MDN compatibility data into RemoteSettings",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "ingest": "node update_remote_settings_records.mjs"
+  },
+  "author": "",
```

Comment:
Not sure how "complete" we should make this package json now that we have it on github. I guess we are not going to publish it, so not sure if a valid author and license are needed.

Reply:
not sure either, I'll check if there's some convention about package.json file for projects which won't be published

Reply:
I removed most of the properties, we can reevaluate if we ever need those in the future